modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -717,9 +717,23 @@ namespace {
             // FIXME: Introduce a DNNBackend interface which'd specify
             // the framework for this???
             GIEModel gm(gr);
-            const auto &np = gm.metadata(nh).get<NetworkParams>();
-            const auto &pp = cv::util::any_cast<cv::gapi::ie::detail::ParamDesc>(np.opaque);
+            auto &np = gm.metadata(nh).get<NetworkParams>();
+            auto &pp = cv::util::any_cast<cv::gapi::ie::detail::ParamDesc>(np.opaque);
             const auto &ki = cv::util::any_cast<KImpl>(ii.opaque);
+
+            GModel::Graph model(gr);
+            auto& op = model.metadata(nh).get<Op>();
+
+            // NB: In case generic infer, info about in/out names is stored in operation (op.params)
+            auto* in_out_info = cv::util::any_cast<cv::InOutInfo>(&op.params);
+            if (in_out_info)
```

Comment:
`cv::util::any` needs to have `has_value` method :)

Reply:
It should be there, isn't it?

Reply:
Anyway, no logic on dynamic cast please.

Reply:
Fixed